docs/source/schema.rst

```diff
@@ -531,3 +531,22 @@ Available events
       "delay": "1h"
     }
   }
+
+Custom Isolation Level
+-------------------------
+
+You can specify transaction isolation level for api request.
```

Comment:
You can specify the (article) transaction isolation level for api requests (plural).